### PR TITLE
use gradlew 4.8 and org.jetbrains.intellij 0.3.4

### PR DESCRIPTION
**Pull request by Eskibear**

### CODE REVIEW

**Comment by Eskibear** [comment]
@wezhang please help to have a look. 
Use the Gradle plugins DSL to resolve travis issue. 

Ref: 
https://kotlinlang.org/docs/reference/using-gradle.html

File: PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle [hunk 1]
```diff
@@ -1,26 +1,7 @@
...
-
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
-}
-
 plugins {
-    id "org.jetbrains.intellij" version "0.3.3"
+    id "org.jetbrains.intellij" version "0.3.4"
+    id "org.jetbrains.kotlin.jvm" version "1.2.50"
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle]: Can we use
```gradle
plugins {
    kotlin("jvm") version "1.2.50"
}
```
?

Comment (Eskibear) [reply PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle]: Yes we can, but from the docs, I don't see any difference between the 2 forms. Just wondering, do you know the differences between the `Gradle plugins DSL` and `Gradle Kotlin DSL`?

Comment (wezhang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle]: Aha, Gradle Kotlin DSL looks more aggressive, it 's 

>  a Kotlin-based approach to writing Gradle build scripts and plugins

Let's just keep the current version, groovy is enough currently.

Comment (Eskibear) [reply PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle]: Ok, it makes sense.

**Pull request merged**